api/routers/player.py

```diff
@@ -6,7 +6,8 @@
 from api.database.database import Engine, EngineType
 from api.database.functions import sqlalchemy_result, verify_token
 from api.database.models import Player as dbPlayer
-from fastapi import APIRouter, HTTPException, Query
+from api.utils.logging_helpers import build_route_log_string
```

Comment:
Google Style guide says: 

> Use import statements for packages and modules only, not for individual classes or functions.

* we make an adaptation that classes can be imported aswell